ext4/superblock.py

```diff
@@ -110,7 +111,7 @@ class Superblock(Ext4Struct):
         ("s_usr_quota_inum", c_uint32),
         ("s_grp_quota_inum", c_uint32),
         ("s_overhead_blocks", c_uint32),
-        ("s_backup_bgs", c_uint32),
+        ("s_backup_bgs", c_uint32 * 2),
```

Comment:
Could you point me at the ext4 doc that describes this.

Reply:
https://kernel.org/doc/html/latest/filesystems/ext4/globals.html#super-block